### PR TITLE
Task502 estimate benefit button

### PR DESCRIPTION
**Pull request by EphraimAssi**
# Description 📝
Added a button for estimating benefits on benefit details page. 

## Taiga Link 🔗
https://taiga.dts-stn.com/project/ericdube-hfp-shared-backlog/task/502?kanban-status=134

## Checklist ✅
- [ ] created story for all visual components
- [ ] created or modified necessary unit tests
- [ ] created or modified e2e tests
- [ ] modified README or any applicable documentation on the changes you made

### CODE REVIEW

File: src/pages/benefit.js [hunk 1]
```diff
@@ -111,6 +111,12 @@ export function BenefitPage() {
           ? benefitData.benefitContent
           : "Looks like there is no content yet"
       }
+      EstimateBenefitText={
+        benefitData.benefitTag.includes("Internal") & keycloak.authenticated
```
Comment (will0684) [inline src/pages/benefit.js]: The operator here should be "&&"

**Review by will0684 — CHANGES_REQUESTED** [review]
image

This error appears when you go to a provincial benefit page.

**Review by shewood — COMMENTED** [review]
Would it be possible to make the button the same size as the Apply button?
Apart from that, it looks great so far.

**Comment by EphraimAssi** [comment]
Fixed the code but couldn't adjust the button size since it varies in English and French.

**Review by shewood — APPROVED** [review]
Error is still there.

**Review by shewood — COMMENTED** [review]
Oh, I do still see the error. No estimates on the provincial pages.

**Pull request closed without merging**